framework/test/unit/modules/interoperability/mainchain/commands/submit_mainchain_cross_chain_update.spec.ts

```diff
@@ -283,6 +284,17 @@ describe('SubmitMainchainCrossChainUpdateCommand', () => {
 		jest.spyOn(bls, 'verifyWeightedAggSig').mockReturnValue(true);
 	});
 
+	describe('verify schema', () => {
```

Comment:
I think its missing validation for `activeValidatorsUpdate.blsKeysUpdate,` `inboxUpdate.messageWitnessHashes` and `inboxUpdate.outboxRootWitness.siblingHashes`